pkg/controller/utils/capability.go

```diff
@@ -45,12 +46,17 @@ const (
 	UsageTag = "+usage="
 	// ShortTag is the short alias annotation
 	ShortTag = "+short"
+	// K8sSecretSchemaTag marks the value should be set as an context
+	K8sSecretSchemaTag = "+k8sSecretSchema="
```

Comment:
I think `+k8sSecretSchema` as keyword is better.

Users can easily make mistakes to write like `+k8sSecretSchema =`, with a space in it. Then will the feature be 
 broken?

Reply:
Done